schema/deploy/util_functions/get_form_status_001.sql

```diff
@@ -26,6 +26,7 @@ $function$
         )
```

Comment:
Got confused by the filename initially--this PR is a rework of a _001 file?

Reply:
yeah, it looks like a _001 file got created at one point instead of a rework, so now I'm just reworking the new file.